go.mod

```diff
@@ -206,6 +206,7 @@ require (
 	github.com/kr/pretty v0.3.0 // indirect
 	github.com/kr/text v0.2.0 // indirect
 	github.com/libdns/libdns v0.2.1 // indirect
+	github.com/lithammer/dedent v1.1.0 // indirect
```

Comment:
How was this dependency added if it's indirect but no direct dependencies has changed?

Reply:
Not sure, I just `go get github.com/lithammer/dedent`. Is it maybe marked indirect because it's only used in test code?

Reply:
Ok, `go mod tidy` seems to now have moved it to direct dependency status.